packages/@react-spectrum/contextualhelp/src/contextualhelp.css

```diff
@@ -30,7 +30,7 @@
 
 }
 
-.react-spectrum-ContextualHelp-dialog {
+.react-spectrum-ContextualHelp-dialog.react-spectrum-ContextualHelp-dialog {
```

Comment:
just curious, what's it conflicting with?

Reply:
.Spectrum-Dialog